internal/impl/amqp09/integration_test.go

```diff
@@ -77,14 +123,41 @@ input:
 		integration.StreamTestSendBatch(10),
 		integration.StreamTestStreamSequential(1000),
 		integration.StreamTestStreamParallel(1000),
-		integration.StreamTestStreamParallelLossy(1000),
-		integration.StreamTestStreamParallelLossyThroughReconnect(1000),
 	)
-	suite.Run(
-		t, template,
-		integration.StreamTestOptSleepAfterInput(500*time.Millisecond),
-		integration.StreamTestOptSleepAfterOutput(500*time.Millisecond),
+
+	// we can't run these tests when auto-delete is not set because the disconnect / reconnect cycle cleans up the queues under test
+	if !setQueueDeclareAutoDelete {
```

Comment:
This has changed since last revision- we don't run the lossy tests while exercising the auto-delete functionality.